.github/workflows/ci.yml

```diff
@@ -0,0 +1,127 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+    - master
+
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+
+    # To enable coverage, go to https://codecov.io/, acquire a token, put it
+    # into your secrets (Settings > Secrets > New repository secret) and delete
+    # the "if" conditional below.
+    # If you do not wish to use codecov, then simply delete this job from the
+    # workflow.
```

Comment:
Cool, will do!

Reply:
I generated a token and make a secret called CODECOV_TOKEN.
That's it?

Reply:
Yes, it's nothing fancy. The comments there describe everything you have to do. I will not change this in my branch, because it's just going to fail anyway.

Reply:
Ok, I can turn it on later!